### DISCUSSION TITLE
关于GPT本地化部署的问题

### DISCUSSION THREAD

**Discussion by RoyHughluo** [Ayanaminn/N46Whisper#23] [General]
请问下Aya老师，这个项目会弄一个本地跑的版本么？这样可以避免colab的付费

**Comment by Ayanaminn** [comment]
可能暂时不会...一方面是因为类似的需求已经有一些本地版实现方案了？比如 https://github.com/m1guelpf/auto-subtitle ，https://github.com/gnehs/subtitle-translator， 其实Whisper本身也支持直接出srt的命令
另一方面是我自己也没有那么高的显存配置，没法调试本地版:joy:

**Reply by RoyHughluo** [reply]
hhh了解了，感谢Aya老师~
